rig/routing_table/__init__.py

```diff
@@ -1,6 +1,7 @@
 from rig.routing_table.entries import RoutingTableEntry, Routes
 from rig.routing_table.exceptions import MinimisationFailedError
-from rig.routing_table.utils import table_is_subset_of, expand_entries
+from rig.routing_table.utils import (table_is_subset_of, expand_entries,
+                                     intersect, remove_default_routes)
```

Comment:
If `remove_default_routes` is "just another routing algorithm", there is no reason to import it into the top-level namespace.